includes/payment-methods/class-wc-stripe-upe-payment-method-afterpay-clearpay.php

```diff
@@ -19,7 +19,7 @@ public function __construct() {
 		$this->stripe_id                    = self::STRIPE_ID;
 		$this->title                        = __( 'Clearpay / Afterpay', 'woocommerce-gateway-stripe' );
 		$this->is_reusable                  = false;
-		$this->supported_currencies         = [ 'USD', 'CAD', 'GBP', 'AUD', 'NZD' ];
+		$this->supported_currencies         = [ WC_Stripe_Currency_Code::UNITED_STATES_DOLLAR, WC_Stripe_Currency_Code::CANADIAN_DOLLAR, WC_Stripe_Currency_Code::POUND_STERLING, WC_Stripe_Currency_Code::AUSTRALIAN_DOLLAR, WC_Stripe_Currency_Code::NEW_ZEALAND_DOLLAR ];
```

Comment:
What do you think about breaking up this now-superlong line into multiple lines?